support/ci/pipelines/build.sh

```
@@ -77,18 +77,13 @@ EOF
 python3 "$SCRIPT_DIR/../../runner-script.py" dependencies prefetch-needed --all --target "$TARGET"
 
 mkdir -p build
-nix build -o build/out-$TARGET -v -L .#all-for-platform.$TARGET --show-trace
+nix build -o build/out-$TARGET -v -L .#$TARGET --show-trace
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)